internal/view/exec.go

```diff
@@ -378,6 +378,9 @@ func k9sShellPod(node string, cfg *config.ShellPod) *v1.Pod {
 	if len(cfg.Args) > 0 {
 		c.Args = cfg.Args
 	}
+	if len(cfg.ImagePullPolicy) > 0 {
+		c.ImagePullPolicy = v1.PullPolicy(cfg.ImagePullPolicy)
```

Comment:
If we set the type to PullPolicy as per comment above, no need to promote the string.